### PR TITLE
restructured and added some html content

### PR DESCRIPTION
**Pull request by lynnsamuelson**

### CODE REVIEW

File: HTML_CSS_FLEXBOX/Exercises/Counting.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <article class="four number">Four</article>
+    <article class="three number">Three</article>
+    <article class="six number">Six</article>
+    <article class="nine number">Nine</article>
+  </section>
+```
+
+### Challenges
+
+Once you've done that, try these challenges.  All still use CSS only and each should build on the others.
```
Comment (simplesnoh) [inline HTML_CSS_FLEXBOX/Exercises/Counting.md:22]: Maybe write it as "they all still use only CSS and each one should build on the others"?

**Review by simplesnoh — CHANGES_REQUESTED** [review]

File: HTML_CSS_FLEXBOX/Content/HTML.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  <li>Apples</li>
+  <li>Grapes</li>
+  <li>Strawberries</li>
+</ul>
+```
+
+
+## Anatomy of an HTML Element
+
+![./Images/flexbox.png](https://github.com/nashville-software-school/client-side-mastery/blob/E20/HTML_CSS_FLEXBOX/Images/HTML_Tag_Anatomy.png)
```
[the remark this answers is not in the document]
Comment (lynnsamuelson) [reply HTML_CSS_FLEXBOX/Content/HTML.md:39]: I made the link work with the E20 branch and the image is not in the correct path yet.

File: HTML_CSS_FLEXBOX/Content/HTML.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+# HTML Components
+
+## Why Are You Learning This?
+
+Thinking of applications and web page as a collection of independent components is the foundation of modern web applications that you use all the time.
```
[the remark this answers is not in the document]
Comment (lynnsamuelson) [reply HTML_CSS_FLEXBOX/Content/HTML.md:5]: ???

File: HTML_CSS_FLEXBOX/Content/HTML.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+Attribute Values: This is the value associated with an attribute.  In the example above the id="nurse".  If we are looking for the element with an id of nurse, this would be that element.  
+
+Content: This is what gets displayed.  
+
+Closing Tag: This indicates the element is now finished.  
+
+## Atributes
+
+- HTML elements can have many different attributes.
+
```
[the remark this answers is not in the document]
Comment (lynnsamuelson) [reply HTML_CSS_FLEXBOX/Content/HTML.md:54]: fixed

File: HTML_CSS_FLEXBOX/Content/HTML.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+More Information: This is not required reading, but good references.
+
+https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes
+
+https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes
+
+## Self Closing Tags
+
+These tags do not require a closing tag.
```
[the remark this answers is not in the document]
Comment (lynnsamuelson) [reply HTML_CSS_FLEXBOX/Content/HTML.md:83]: fixed

File: HTML_CSS_FLEXBOX/Content/HTML.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+
+## Common HTML elements include:
+
+Block: an element that occupies 100% of the available width has a line break before and after the element
+```html
+<div>, <p>, <h1>, <h2>, <h3>, <h4>, <h5>, <h6>, <form>, <ol>, <ul>, <li>,  
+```
+
+Inline:an element that takes up as much space as it needs or is styled to take-up.
```
[the remark this answers is not in the document]
Comment (lynnsamuelson) [reply HTML_CSS_FLEXBOX/Content/HTML.md:102]: fixed

**Pull request merged**